conda_devenv/devenv.py

```diff
@@ -597,6 +602,34 @@ def main(args=None):
         print(f'File "{filename}" does not exist.', file=sys.stderr)
         return 1
 
+    env_manager = args.env_manager
+    if env_manager is None:
+        if "CONDA_DEVENV_ENV_MANAGER" not in os.environ:
+            print("You must especify and enviroment manager.", file=sys.stderr)
+            print(
+                (
+                    'Use the "--env-manager" ("-m") flag or the '
+                    '"CONDA_DEVENV_ENV_MANAGER" environment variable.'
+                ),
+                file=sys.stderr,
+            )
+            return 1
```

Comment:
We should default to using `conda`, and not force every user to define the envvar or use `-m`... after all the name of the project is CONDA-devenv.

Reply:
Agree, but I still think we should make `conda-devenv` use conda by default... I guess we need to make very explicit in the docs how mamba support is meant to be used.